docs/user_manual/boxes.md

```diff
@@ -0,0 +1,328 @@
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% assign figure = figure | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Adding Multiple Items to a Box" %}
+
+To add multiple items to a box, go to the Edit Box page by clicking on the box name in the Boxes list page.
+Select the target positions in the Contents diagram. Find the Position and Search rows adjacent to the
+Contents diagram (see figure {{figure}}), and enter the target item's name, alias, or barcode into the Search
+field for the target position. 
+
+{% include userman-figure.md num=figure cap="Additional information when multiple positions are selected"
+img="multiple-positions-selected.png" %}
+
+Once all Search fields are filled out, click "Update" below the last search field. If the target
+position already contains an item, a dialog box will name the items that already in those positions and ask
+you to confirm that you wish to replace them with the items matching your search terms. If any items to be
+replaced are not being re-added to a different position in the box, they will be removed from the box. Click
+"Replace" to continue, or "Cancel" to cancel the update and select new positions.
+
+If one item is found for each position, the page is refreshed and the Contents diagram and table will contain
+the target items. If more than one item is found for a given position, or if no matching item can be found for
+a given search term, no items will be updated and a dialog box will display the search terms which were in
+error.
+
+Multiple items can also be added to a box using the [Scanning a box](#scanning_a_box) instructions.
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Removing a Single Item from a Box" %}
+
+A single item can be removed from a box, which removes its Location value. It is advised to scan it into
+another box or location position soon in order to keep tracking its location in MISO.
+
+To remove a single item from a box, go to the Edit Box page by clicking on the box name in the Boxes list
+page. Select the target position in the Contents diagram. In the information section that appears adjacent to
+the diagram, click "Remove Item". In the dialog box that appears, click "Remove" to confirm that you wish to
+remove this item from the box. The page will refresh, and the target position will be empty in the Contents
+diagram.
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Discarding a Single Item from a Box" %}
+
+A single item can be discarded from a box to indicate that the item no longer needs to be tracked. When an
+item is discarded, its volume will be set to "0", discarded will be set to True, and its location will be
+shown as "EMPTY". Discarded items can not be added to boxes.
```

Comment:
On the Edit Sample page? I don't see "EMPTY" anywhere - either for location or box location

Reply:
Interesting - it shows as EMPTY on the List Samples page. I'll file a ticket to harmonize these.